pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+line-length = 100
```

Comment:
I think black will need to be in a different PR, if we do this now, zuul will run and notice many files need to be changed.  The line length can go in when black is added to pre-commit